client/src/modules/Auth/SignIn/actions.js

```diff
@@ -0,0 +1,48 @@
+import {
+  SIGNIN_REQUEST,
+  SIGNIN_SUCCESS,
+  SIGNIN_FAILURE
+} from './const';
+
+import signInUser from './api';
+
+function request(user) {
+  return {
+    type: SIGNIN_REQUEST,
+    payload: user
+  };
+}
```

Comment:
You receive user payload, but You don't use it in signIn action. In reducer that action type return loading. 
If you need loading, check setLoading action in ItemList actions.